content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
+You can also apply for a [student loan](/funding-your-training#tuition-fee-and-maintenance-loans).
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a school direct <span>course</span></a>
-</p>
+<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find university training</a>
 
-$postgraduate-teaching-apprenticeship$
+### Learn on the job in schools near you
 
-You’ll be employed by a school and learn while you’re working.
+If you already have a degree and want to spend as much time as possible in practical learning, you can find a training programme led by a school or group of schools.
 
-You’ll split your time between school and university study, with at least 20% of your time studying.
+You'll only spend about about a fifth of your time in academic learning (for example, written assignments).
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a postgraduate teaching <span>apprenticeship</span></a>
-</p>
+Many courses even give you a PGCE or PGDE - an academic qualification which you can use towards masters credits.
 
-$teach-first$
+Courses typically take one year but you may be able to study part time.
 
-The Teach First Leadership Development Programme is a two-year salaried programme.
+Although you often have to pay course fees of around £9,250 you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.
 
-It combines leadership development with teacher training, and focuses on training exceptional teachers to teach in challenging schools.
+You can also apply for a [student loan](/funding-your-training).
 
-You’ll learn on the job while you work towards Qualified Teacher Status (QTS) and a Postgraduate Diploma in Education (PGDE).
+Some training programmes offer a salary, although these can be harder to find.
 
-A PGDE contains up to 120 credits at master’s degree level.
+<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find courses near you</a>
 
-<p class="call-to-action__action">
-  <a href="https://www.teachfirst.org.uk/">Find a Teach First <span>course</span></a>
-</p>
+#### Teach First
 
-$undergraduate$
+You can also consider [Teach First](https://www.teachfirst.org.uk/). It's a salaried, 2-year programme leading to a PGDE and qualified teacher status.
 
- You can get Qualified Teacher Status (QTS) as part of an undergraduate degree course.
- 
- This could be right for you if you do not have a degree yet.
- 
- You can also [get funding](/funding-your-training).
- 
- <p class="call-to-action__action">
-   <a href="https://digital.ucas.com/search">Find an undergraduate <span>course</span></a>
- </p>
+### If you do not have a degree
 
-$assessment-only$
+You can get qualified teacher status as part of an undergraduate degree.
```

Comment:
Is it style/policy to call these 'undergraduate' rather than 'bachelor' degrees? I think elsewhere on site (e.g. TTA rules) we call it bachelor but I'm ignorant on style guide apologies ...

Reply:
we don't have a style guide yet!